GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -379,9 +379,9 @@ private void chkUseFromOnto_CheckedChanged(object sender, EventArgs e)
 
         private void btnChooseFromRevision_Click(object sender, EventArgs e)
         {
-            bool previousValueBranchFilterEnabled = AppSettings.BranchFilterEnabled;
-            bool previousValueShowCurrentBranchOnly = AppSettings.ShowCurrentBranchOnly;
-            bool previousValueShowReflogReferences = AppSettings.ShowReflogReferences;
+            bool previousValueBranchFilterEnabled = AppSettings.BranchFilterEnabled.Value;
```

Comment:
We need to provide the implicit conversion to/from bool to avoid changes like this one, and will make the changeset much smaller and cleaner.

Reply:
Done for "to `T`".
I would say it is impossible for "from `T`" because `operator =` cannot be overloaded (in contrast to C++) and I cannot create full-blown `RuntimeSettings<T>` from a bool or enum value.